gulpfile.js

```diff
@@ -31,12 +31,7 @@ var paths = {
 
 gulp.task('html', function() {
   return gulp.src('src/index.html')
-    .pipe(data(function(file, callback) {
-      // wrap in {events: … } for template
-      eventData('events', function(err, data) {
-        callback(err, data ? {events: data} : null)
-      })
-    }))
+    .pipe(data(eventData('events/2016/*.md')))
```

Comment:
Is it worth doing a search for the latest year directory so we don't need to change this next year?